app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/adapter/EditItemAdapter.java

```diff
@@ -40,10 +46,27 @@ void updateImageList(List<Tag> imageList) {
         this.tagsList = imageList;
     }
 
+    void setOnTagActionListener(OnTagActionListener listener) {
+        this.onTagActionListener = listener;
+    }
+
+    void clearTag(Tag tag) {
+        if (!tagsList.isEmpty()) {
+            for (int index = 0; index < tagsList.size(); index++) {
```

Comment:
Maybe we should store models in some kind of map to  have easier and quicker lookups